pkg/client/group_mgmt_client.go

```diff
@@ -33,6 +33,7 @@ type DataWrapper struct {
 	StartRow      *int        `json:"startRow,omitempty"`
 	EndRow        *int        `json:"endRow,omitempty"`
 	PageSize      *int        `json:"pageSize,omitempty"`
```

Comment:
No, let's keep this for now in case if we need this in the future . Anyways this is harmless.

Reply:
It's harmless as you say, but I'd like to understand your perspective.  This structure is meant to hold the Nimble REST API payload, right?  I don't think the Nimble API defines this attribute, so I'm having trouble imagining when it might become useful.  It seems confusing to keep it.